<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HastaBilgi</class>
 <widget class="QDialog" name="HastaBilgi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KAYIT MENÜSÜ</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/qtresimler/arkaplan.png</normaloff>../../Desktop/qtresimler/arkaplan.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #12F8DE, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_sira">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>SIRA:      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_sira">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labe_ad">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>AD:         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ad">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_soyad">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SOYAD:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_soyad">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_tc">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TC:         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_tc">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_doktor">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>DOKTOR:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_doktor">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_klinik">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>KLİNİK:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_klinik">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">QTableView{
border: 2px solid green;
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_Temiz">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightgreen;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border: 2px solid green;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Satır Temizle</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_sec_status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightgreen;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border: 2px solid green;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Kaydet</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_sil">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightgreen;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border: 2px solid green;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Hasta Kaydını Sil</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_edit">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightgreen;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border: 2px solid green;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Hasta Güncelle</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_arama">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightgreen;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border: 2px solid green;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Hasta Ara</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="txt_arama">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightblue;
    selection-background-color: darkgray;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>TC Giriniz.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_goster">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightgreen;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border: 2px solid green;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Kayıtlı Hastaları Göster</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_temiz">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
